Improve Getting Started (#229)
meilisearch/documentation: guides/introduction/quick_start_guide.md

::: tab JS

```bash
npm install meilisearch
# or
yarn add meilisearch
```

```js
const MeiliSearch = require("meilisearch");

var client = new MeiliSearch({ host: "http://127.0.0.1:7700" });
const index = client

>   .createIndex({ uid: "indexUID" })

Reviewer: For consistency between the examples
```suggestion
  .createIndex({ uid: "movies" })
```

  .then((res) => console.log(res));
```

[About this package](https://github.com/meilisearch/meilisearch-js/)
:::

::: tab Ruby

```bash
$ gem install meilisearch
```

```ruby
require 'meilisearch'

client = MeiliSearch::Client.new('http://127.0.0.1:7700')
index = client.create_index('movies')
```

[About this package](https://github.com/meilisearch/meilisearch-ruby/)
:::

::: tab PHP

```bash
$ composer require meilisearch/meilisearch-php
```

```php
<?php

require_once __DIR__ . '/vendor/autoload.php';

use MeiliSearch\Client;

> $client = new Client('http://localhost:7700');

Reviewer: ```suggestion
$client = new Client('http://127.0.0.1:7700');
```

$index = $client->createIndex('movies');
```

[About this package](https://github.com/meilisearch/meilisearch-php/)
:::

::: tab Python

```bash
$ pip3 install meilisearch
```

```python
import meilisearch

client = meilisearch.Client('http://127.0.0.1:7700')
index = client.create_index(uid='movies')
```

[About this package](https://github.com/meilisearch/meilisearch-python/)
:::

::: tab Go

```bash
go get github.com/meilisearch/meilisearch-go
```

```go
package main

import (
    "github.com/meilisearch/meilisearch-go"
)

func main() {
    var client = meilisearch.NewClient(meilisearch.Config{

>       Host: "http://localhost:7700",

Reviewer: ```suggestion
      Host: "http://127.0.0.1:7700",
```

```

[About this package](https://github.com/meilisearch/meilisearch-js/)
:::

::: tab Ruby

```ruby
require 'json'

movies_json = File.read('movies.json')
movies = JSON.parse(movies_json)
index.add_documents(movies)
```

[About this package](https://github.com/meilisearch/meilisearch-ruby/)
:::

::: tab PHP

```php
$movies_json = file_get_contents('movies.json');
$movies = json_decode($movies_json);
$index->addDocuments($movies);
```

[About this package](https://github.com/meilisearch/meilisearch-php/)
:::

::: tab Python

```python
import json

json_file = open('movies.json')
movies = json.load(json_file)
index.add_documents(movies)
```

[About this package](https://github.com/meilisearch/meilisearch-python/)
:::

::: tab Go

```go
import (
    "encoding/json"
    "io/ioutil"
)
```

```go
moviesJSON, _ := os.Open("movies.json")
defer moviesJSON.Close()

byteValue, _ := ioutil.ReadAll(moviesJSON)
var movies []map[string]interface{}
json.Unmarshal(byteValue, &movies)

updateRes, _ := client.Documents("movies").AddOrUpdate(movies)
fmt.Println(updateRes.UpdateID)
```

[About this package](https://github.com/meilisearch/meilisearch-go/)
:::

::::

### Checking updates

> The document addition returns a simple JSON with only an `updateId`.

Reviewer: I don't think simple is a useful adjective. And a "JSON" is a format not a type, so I added object. It is possible I made the same mistake somewhere else in the documentation.
```suggestion
The document addition returns a JSON object with only an `updateId` attribute.
```

This kind of **successful response** indicates that the operation has been taken into account, but it may not have been executed yet.

You can check the status of the operation via the `updateId` and the [get update status route](/references/updates.md).

> Checking the update status is not a mandatory step to search through your documents but could be really useful in case of error.

Reviewer: ```suggestion
Checking the update status is not a mandatory step to search through your documents but could be really useful to understand what is happening in case of errors or unexpected behaviors.
```

      "overview": "ve Victorian Age Gotham City, Batman begins his war on crime",
      "release_date": "2018-01-12"
    }
    ...
  ],
  "offset": 0,
  "limit": 20,
  "processingTimeMs": 2,
  "query": "botman"
}
```

### Web Interface

MeiliSearch also offers an out-of-the-box [web interface](/guides/advanced_guides/web_interface.md) on which you can try the search.

> All you need to do is open your web browser and enter MeiliSearch’s address to visit it locally (in our case: `http://127.0.0.1:7700`).<br>

Reviewer: ```suggestion
All you need to do is open your web browser and enter MeiliSearch’s address to visit it (in our case: `http://127.0.0.1:7700`).<br>
```

> This will lead you to a web page with a search bar that will allow you to search in a selected index.

Reviewer: ```suggestion
This will lead you to a web page with a search bar that will allow you to search in the selected index.
```
